autodistill/cli.py

```diff
@@ -0,0 +1,167 @@
+import json
+import os
+
+import click
+import cv2
+import roboflow
+import supervision as sv
+
+from autodistill.detection.caption_ontology import CaptionOntology
+from autodistill.registry import import_requisite_module
+
+SUPPORTED_ROBOFLOW_MODEL_UPLOADS = ["yolov5", "yolov5-seg", "yolov8", "yolov8-seg"]
+
+SUPPORTED_MODEL_TYPES = ["detection", "segmentation" "classification"]
+
+SUPPORTED_DATASET_FORMATS = ["yolov8", "yolov5", "voc"]
+
+
+@click.command()
+@click.argument("images")
```

Comment:
Should we set the `default` to the current directory?

We should probably pass the `help=` param to these also to describe what they are/do.